dockers/docker-orchagent/switch.json.j2

```diff
@@ -18,13 +18,15 @@
 [
     {
         "SWITCH_TABLE:switch": {
+{% if DEVICE_METADATA.localhost.switch_type != "dpu" %}
```

Comment:
Can you add modify to this -` {% if DEVICE_METADATA.localhost.switch_type and DEVICE_METADATA.localhost.switch_type != "dpu" %}`

Reply:
By default, this configuration should be enabled. We want to disable it only for DPU.
The `switch_type` attribute in DEVICE_METADATA table is optional. When the attribute is missing the assumption is that the type is `switch`. The condition for this should be the following:
`{% if not DEVICE_METADATA.localhost.switch_type or DEVICE_METADATA.localhost.switch_type != "dpu" %}`
@prsunny can you please confirm this?

Reply:
@oleksandrivantsiv , I feel thats still not going to achieve the intended functionality with 'or'. One condition will be true when swtich_type is present even with 'dpu' and continues with setting attributes.

Reply:
The condition is 'if **not** DEVICE_METADATA.localhost.switch_type or DEVICE_METADATA.localhost.switch_type != "dpu"'. 

It should hit in two cases. When the `switch_type` is not present in the table:
<img width="1318" alt="image" src="https://github.com/sonic-net/sonic-buildimage/assets/17766225/9afcde1a-61bd-42dd-8f42-cb735b40232e">
Or when `switch_type` is not equal to `dpu`
<img width="1318" alt="image" src="https://github.com/sonic-net/sonic-buildimage/assets/17766225/4b3be9f8-4f8e-4d6d-a55a-49b620369327">
And should not hit when the `switch_type` is `dpu`
<img width="1305" alt="image" src="https://github.com/sonic-net/sonic-buildimage/assets/17766225/82a650bc-d092-4da4-b8fa-dd744722595e">